Setting sub-identities using Ledger

DrW3RK opened the pull request:
PR for this issue - https://github.com/w3f/polkadot-wiki/issues/3106

DrW3RK committed:
Setting sub-identities using Ledger

DrW3RK committed:
Update learn-identity.md

DrW3RK committed:
minor fix

DrW3RK committed:
minor fix

laboon reviewed (changes requested):
We should also include this on https://wiki.polkadot.network/docs/kusama-ledger

I would prefer to see this information in text form there (step by step instructions like https://wiki.polkadot.network/docs/kusama-ledger#removing-expired-democracy-locks) and then linking to the video as supplemental information.

DrW3RK committed:
Step-by-step Instructions

DrW3RK committed:
updated links to Wiki docs

DrW3RK committed:
fix broken ref

DrW3RK commented:
@laboon Here are the changes made -

- Added step-by-step text instructions for setting sub-identity
- Added relevant screenshots
- Updated the instructions on Sub Accounts wiki section with screenshots and video link

DrW3RK committed:
fix boroken link

DrW3RK committed:
Merge branch 'sub-id-ledger' of https://github.com/w3f/polkadot-wiki into sub-id-ledger

laboon commented on docs/general/kusama/kusama-ledger.md:
```diff
@@ -228,6 +228,28 @@ account stored on a Ledger device, as follows:
+  ![Set sub-identity in PolkadotJS](../../assets/identity/sub-id-2.png)
+- Sign and submit the transaction from the parent account with the identity
+
+You should now see the sub-identity displayed on-chain. You need to aware that 0.033 KSM is reserved for setting 
```
"0.033" and "0.006" should be variables, not hard-coded.

Otherwise looks good to me!

DrW3RK committed:
Variables instead of hardcoding
